modules/monitoring/files/grafana-webhook.py

```diff
@@ -20,11 +20,20 @@ def post():
         with lock:
             lock.acquire()
             try:
-                message = content['state'] + ' : ' + content['title']
-                if content['alerts'][0]['labels']['team'] == 'mediawiki':
-                    message = message + ' https://grafana.miraheze.org/d/GtxbP1Xnk/mediawiki'
                 x = open('/var/log/icinga2/irc.log', 'a+')
```

Comment:
Nit: I probably introduced this but while we are cleaning up can we change x to a nicer name?

Reply:
Done